share/spice/hex/content.handlebars

```diff
@@ -0,0 +1,8 @@
+<p><strong>{{releases.0.version}}</strong> is the latest of <strong>{{releases.length}}</strong> releases and it was published <em>{{updated_at}}</em>.</p>
```

Comment:
We don't have our own set of rules written quite yet. So far, we haven't
been strict about enforcing a universal style but in our docs we do say
that we generally follow Crockford's style/conventions.
 
I was interested in using a service like Hound CI to provide feedback on
pull requests (mostly so devs don't have to remember to run Gulp etc)
but it doesn't seem to support Perl or CSS.

I looked into Codacy a long time ago and it seems to have come a long
way so that might be another option.
 
Are you aware of any other hosted solutions that might work like that?
 
--
Zaahir Moolla
DuckDuckGo[1]
 
 
On Mon, 18 Apr 2016, at 01:29 PM, Milton Mazzarri wrote:
> In share/spice/hex/content.handlebars[2]:


> > @@ -0,0 +1,8 @@ > +<p><strong>{{releases.0.version}}</strong> is the
> > latest of <strong>{{releases.length}}</strong> releases and it was
> > published <em>{{updated_at}}</em>.</p>
>
> @moollaza[3] Thanks for your advice, already applied your suggestion
> and bookmarked the link :bowtie:
> May I ask if you have some guidelines for JavaScript code? I have not
> seen any Task Manager (e.g. Gulp[4]) in the zeroclickinfo-spice
> project to run a static code analysis tool (e.g. JSHint) or verify
> that our JS code complies with the DuckDuckGo's JS style guide, using
> something like JSCS[5].
> Thanks in advance for your help.
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub[6]
>
 

Links:

  1. https://duckduckgo.com
  2. https://github.com/duckduckgo/zeroclickinfo-spice/pull/2690#discussion_r60099226
  3. https://github.com/moollaza
  4. http://gulpjs.com/
  5. http://jscs.info/
  6. https://github.com/duckduckgo/zeroclickinfo-spice/pull/2690/files/3d2ed93be1b795ab9538331f31c2ae8f004636a7#r60099226